en/Obsidian Sync/Select files and settings to sync.md

```diff
@@ -29,35 +47,50 @@ Common examples of hidden folders that will not be synced include `.vscode`, `
 
 ### Sync settings are not synced
 
-Sync settings are not synced so that each each device can be separately configured according to your needs.
-
-> [!tip] Restart the app to apply settings changes on other devices
-> Obsidian **only** applies vault settings when the app starts. If you change a setting on one device, you **must restart** Obsidian on your other devices for the new changes to take effect. 
-> 
-> For example, if you change the path of your daily notes in the [[Daily Notes]] plugin, you need to restart Obsidian on your other devices to use the new path.
+Sync settings themselves are not synced so that each device can be separately configured according to your needs.
```

Comment:
This is a pretty good tongue twister.

I wonder if we can avoid the reusing the word `sync` as the verb here to make this a bit less confusing.

Here's my attempt:

> Sync settings are not shared across devices. You must Each device must separately configure the settings on each device according to your needs.

Reply:
I think you may have missed a word?

Was it meany to be the below?
> Sync settings are not shared across devices. You must separately configure the settings on each device according to your needs.

Reply:
oh oops, yea I just didn't delete some words in the edit. That's what I meant.

Reply:
> oh oops, yea I just didn't delete some words in the edit. That's what I meant.

Great, I'll push an update. You fine with me merging after this?